src/PostgREST/Config/Database.hs

```diff
@@ -29,6 +30,7 @@ import Protolude
 
 type RoleSettings     = (HM.HashMap ByteString (HM.HashMap ByteString ByteString))
 type RoleIsolationLvl = HM.HashMap ByteString SQL.IsolationLevel
+type TimezoneNames    = [Text] -- cache timezone names for prefer timezone=
```

Comment:
@taimoorzaeem How about turning this into a [Set](https://hackage.haskell.org/package/containers-0.7/docs/Data-Set.html). I think you only need to apply `member` over it. Should be faster than a List since it uses balanced trees.